test/bundler/esbuild/bundler-test.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+
+const t = require('tap')
+const test = t.test
+const fastifySuccess = require('./dist/success')
+const fastifyFailPlugin = require('./dist/failPlugin')
+
+test('Bundled package should work', (t) => {
+  t.plan(4)
+  fastifySuccess.ready((err) => {
+    t.error(err)
+    fastifySuccess.inject(
+      {
+        method: 'GET',
+        url: '/'
+      },
+      (error, res) => {
+        t.error(error)
+        t.equal(res.statusCode, 200)
+        t.same(res.json(), { hello: 'world' })
+      }
+    )
+  })
+})
+
+test('Bundled package should not work with bad plugin version', (t) => {
+  t.plan(1)
+  fastifyFailPlugin.ready((err) => {
+    t.ok(err)
```

Comment:
I would check using `t.match(err.message, /something/i)` to be sure that the error is the one expected.

It happened that an error has changed in past and we did not spot it